[Common] (terminology) - Standardize use of terms like 'Common API' and 'host-specific API'

kbrandl opened the pull request.

o365devx commented:
OPS Build status updates of commit _[4496614](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/4496614db1ca58a0180458a1e6fea6059b817760)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

o365devx commented:
OPS Build status updates of commit _[4496614](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/4496614db1ca58a0180458a1e6fea6059b817760)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

o365devx commented:
OPS Build status updates of commit _[4496614](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/4496614db1ca58a0180458a1e6fea6059b817760)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/install-latest-office-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/install-latest-office-version.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/understanding-the-javascript-api-for-office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/understanding-the-javascript-api-for-office.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-tutorial-open-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/excel-tutorial-open-dialog.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/javascript-api-for-office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/javascript-api-for-office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/preview-requirement-set/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/preview-requirement-set/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.1/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.1/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.1/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.2/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.2/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.2/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.3/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.3/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.4/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.4/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.5/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.5/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.6/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.6/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.7/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/objectmodel/requirement-set-1.7/office.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/add-in-commands-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/add-in-commands-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/dialog-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/dialog-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/office-add-in-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/office-add-in-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/onenote-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/onenote-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/powerpoint-and-project-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/powerpoint-and-project-note.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/word/word-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/word/word-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812271947212858-649%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=erBRZJEjBX3d4WifQuf825S7qpxgNdDG%2BMlJsIvyID4%3D&st=2018-12-27T19%3A43%3A55Z&se=2019-01-27T19%3A48%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

AlexJerabek commented on docs/develop/understanding-the-javascript-api-for-office.md:
```diff
@@ -161,7 +161,7 @@ For more information, see [Office.initialize Event](https://docs.microsoft.com/j
 ## Office JavaScript API object model
 
-Once initialized, the add-in can interact with the host (e.g. Excel, Outlook). The [Office JavaScript API object model](office-javascript-api-object-model.md) page has more details on specific usage patterns. There is also detailed reference documentation for both [shared APIs](https://docs.microsoft.com/office/dev/add-ins/reference/javascript-api-for-office?view=office-js) and specific hosts.
+Once initialized, the add-in can interact with the host (e.g. Excel, Outlook). The [Office JavaScript API object model](office-javascript-api-object-model.md) page has more details on specific usage patterns. There is also detailed reference documentation for both [Common APIs](https://docs.microsoft.com/office/dev/add-ins/reference/javascript-api-for-office?view=office-js) and host-specific APIs ...
```
Unrelated, but can you strip off the "?view-office-js" from the URL?

kbrandl replied:
Sure, I'll make this change globally throughout the doc set.

AlexJerabek commented on docs/reference/javascript-api-for-office.md:
```diff
@@ -8,9 +8,9 @@ ms.date: 10/09/2018
+- **Common APIs** - APIs that were introduced with **Office 2013**. This is loaded for **all Office host applications** and connects your add-in application with the Office client application. The object model contains APIs that are specific to Office clients, and APIs that are applicable to multiple Office client host applications. All of this content is under **Common API**. 
 
-  **Outlook** also uses the common API syntax. Everything under the alias Office contains objects you can use to write scripts that interact with content in Office documents, worksheets, presentations, mail items, and projects from your Office Add-ins. You must use these common APIs if your add-in will target Office 2013 and later. This object model uses callbacks.
+  **Outlook** also uses the Common API syntax. Everything under the alias Office contains objects you can use to write scripts that interact with content in Office documents, worksheets, presentations, mail items, and projects from your Office Add-ins. You must use these Common APIs if your add-in will target Office 2013 and later. This object model uses callbacks.
```
Should the line "This object model uses callbacks" be under the preceding section instead of (or in addition to) the Outlook section?

kbrandl replied:
Sure; I'll add it to the preceding section as well.

AlexJerabek commented on docs/reference/requirement-sets/excel-api-requirement-sets.md:
```diff
@@ -797,9 +797,9 @@ New methods:
+### Settings API in the Excel namespace
 
-[Setting](/javascript/api/excel/excel.setting) object represents a key-value pair of a setting persisted to the document. Now, we've added settings related APIs under Excel namespace. This doesn't offer net new functionality - however this make easy to remain in the promise based batched API syntax reduce the dependency on common API for Excel related tasks.
+The [Setting](/javascript/api/excel/excel.setting) object represents a key-value pair of a setting persisted to the document. Now, we've added settings-related APIs under the Excel namespace. These APIs don't provide net new functionality, since you were previously able to use the Common API to manage document settings. However, by using these APIs in the Excel namespace, you can use promise-base ...
```
I'm not sure how valuable updating this section actually is, but there's a lot of text here for "The Setting object represents a key-value pair of a setting persisted to the document. The functionality of `Excel.Setting` is equivalent to `Office.Settings`, but uses the batched API syntax, rather than the Common API's callback model."

kbrandl replied:
My main reason for editing this section was to eliminate typos like "this make easy to" and to clean up the grammar and spelling/punctuation. Happy to revise the wording per your suggestion.

AlexJerabek commented on docs/word/word-add-ins-programming-overview.md:
```diff
@@ -47,9 +47,9 @@ Write your add-in once, and it will run in all versions of Word across multiple
+You can use two sets of JavaScript APIs to interact with the objects and metadata in a Word document. The first is the [Common API](../reference/javascript-api-for-office.md), which was introduced in Office 2013. Many of the objects in the Common API can be used in add-ins hosted by two or more Office clients. This API uses callbacks extensively.
 
-The second is the [Word JavaScript API](https://docs.microsoft.com/office/dev/add-ins/reference/overview/word-add-ins-reference-overview?view=office-js). This is a strongly-typed object model that you can use to create Word add-ins that target Word 2016 for Mac and Windows. This object model uses promises, and provides access to Word-specific objects like [body](https://docs.microsoft.com/javascr ...
+The second is the [Word JavaScript API](../reference/overview/word-add-ins-reference-overview.md). This is a strongly-typed object model that you can use to create Word add-ins that target Word 2016 for Mac and Windows. This object model uses promises, and provides access to Word-specific objects like [body](/javascript/api/word/word.body?view=office-js), [content controls](/javascript/api/word/w ...
```
Please remove the "?view=office-js" from the URLs.

kbrandl replied:
Done.

o365devx commented:
OPS Build status updates of commit _[4875e25](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/4875e2577b0e0611e47d964aa2e91e1e2e509f83)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

o365devx commented:
OPS Build status updates of commit _[4875e25](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/4875e2577b0e0611e47d964aa2e91e1e2e509f83)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

o365devx commented:
OPS Build status updates of commit _[4875e25](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/4875e2577b0e0611e47d964aa2e91e1e2e509f83)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/concepts/add-in-development-lifecycle.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/add-in-development-lifecycle.md) | :white_check_mark:Succeeded | |
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-color.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/add-in-color.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/design/authentication-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/authentication-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/add-ins-with-angular2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/add-ins-with-angular2.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-addin-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/create-addin-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/install-latest-office-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/install-latest-office-version.md) | :white_check_mark:Succeeded | |
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/develop/localization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/localization.md) | :white_check_mark:Succeeded | |
[docs/develop/office-javascript-api-object-model.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-javascript-api-object-model.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/persisting-add-in-state-and-settings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/persisting-add-in-state-and-settings.md) | :white_check_mark:Succeeded | |
[docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md) | :white_check_mark:Succeeded | |
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/understanding-the-javascript-api-for-office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/understanding-the-javascript-api-for-office.md) | :white_check_mark:Succeeded | |
[docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-core-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-error-handling.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-error-handling.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-tables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-tables.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-tutorial-open-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/excel-tutorial-open-dialog.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812280001056425-649%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QZo716XP0qkG4XdJeL%2FmIQWoO7%2FrZtoLGEy8V%2BbuWXc%3D&st=2018-12-27T23%3A57%3A48Z&se=2019-01-28T00%3A02%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
OPS Build status updates of commit _[907d232](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/907d232f3d3d8bfc621353927ede2f6791e80cb9)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

o365devx commented:
OPS Build status updates of commit _[907d232](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/907d232f3d3d8bfc621353927ede2f6791e80cb9)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

o365devx commented:
OPS Build status updates of commit _[907d232](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/907d232f3d3d8bfc621353927ede2f6791e80cb9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/concepts/add-in-development-lifecycle.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/add-in-development-lifecycle.md) | :white_check_mark:Succeeded | |
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-color.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/add-in-color.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/design/authentication-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/authentication-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/add-ins-with-angular2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/add-ins-with-angular2.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-addin-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/create-addin-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/install-latest-office-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/install-latest-office-version.md) | :white_check_mark:Succeeded | |
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/develop/localization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/localization.md) | :white_check_mark:Succeeded | |
[docs/develop/office-javascript-api-object-model.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-javascript-api-object-model.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/persisting-add-in-state-and-settings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/persisting-add-in-state-and-settings.md) | :white_check_mark:Succeeded | |
[docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md) | :white_check_mark:Succeeded | |
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/understanding-the-javascript-api-for-office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/understanding-the-javascript-api-for-office.md) | :white_check_mark:Succeeded | |
[docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-core-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-error-handling.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-error-handling.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-tables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-tables.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-tutorial-open-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/excel-tutorial-open-dialog.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812280003359110-649%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zBIqNDMhCXLdeiW9pAF8PB9WJ0Ec%2BGLYUmkYzab1Pd0%3D&st=2018-12-28T00%3A00%3A08Z&se=2019-01-28T00%3A05%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

o365devx commented:
OPS Build status updates of commit _[690a317](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/690a31751463a9a3d9d1fe41a52473a158a6b5cf)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

o365devx commented:
OPS Build status updates of commit _[690a317](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/690a31751463a9a3d9d1fe41a52473a158a6b5cf)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

o365devx commented:
OPS Build status updates of commit _[690a317](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits/690a31751463a9a3d9d1fe41a52473a158a6b5cf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/concepts/add-in-development-lifecycle.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/add-in-development-lifecycle.md) | :white_check_mark:Succeeded | |
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-color.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/add-in-color.md) | :white_check_mark:Succeeded | |
[docs/design/add-in-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/add-in-commands.md) | :white_check_mark:Succeeded | |
[docs/design/authentication-patterns.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/authentication-patterns.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/add-ins-with-angular2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/add-ins-with-angular2.md) | :white_check_mark:Succeeded | |
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/automatically-open-a-task-pane-with-a-document.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/automatically-open-a-task-pane-with-a-document.md) | :white_check_mark:Succeeded | |
[docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/bind-to-regions-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-addin-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/create-addin-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-office-add-ins-for-the-ipad.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/develop-office-add-ins-for-the-ipad.md) | :white_check_mark:Succeeded | |
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/install-latest-office-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/install-latest-office-version.md) | :white_check_mark:Succeeded | |
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/develop/localization.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/localization.md) | :white_check_mark:Succeeded | |
[docs/develop/office-javascript-api-object-model.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-javascript-api-object-model.md) | :white_check_mark:Succeeded | |
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/develop/persisting-add-in-state-and-settings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/persisting-add-in-state-and-settings.md) | :white_check_mark:Succeeded | |
[docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/read-and-write-data-to-the-active-selection-in-a-document-or-spreadsheet.md) | :white_check_mark:Succeeded | |
[docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md) | :white_check_mark:Succeeded | |
[docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/understanding-the-javascript-api-for-office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/understanding-the-javascript-api-for-office.md) | :white_check_mark:Succeeded | |
[docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md) | :white_check_mark:Succeeded | |
[docs/excel/co-authoring-in-excel-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/co-authoring-in-excel-add-ins.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-advanced-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-advanced-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-core-concepts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-core-concepts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-error-handling.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-error-handling.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-overview.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-tables.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/excel/excel-add-ins-tables.md) | :white_check_mark:Succeeded | |
[docs/includes/excel-tutorial-open-dialog.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/excel-tutorial-open-dialog.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-programming-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-common-api-term/docs/onenote/onenote-add-ins-programming-overview.md) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C28%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812280012238385-649%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2F9vSGuOXuO91LWYNYPHUV9u7SYOHfv%2Bbb7lD3AXFcNo%3D&st=2018-12-28T00%3A09%3A00Z&se=2019-01-28T00%3A14%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

kbrandl commented:
@AlexJerabek thanks for the review! I've addressed your feedback (via 3 separate [commits](https://github.com/OfficeDev/office-js-docs-pr/pull/649/commits)) -- if you're interested in seeing the changes I've made, I'd suggest looking at each of the 3 most recent commits instead of looking at all files changed by this PR. (Lots of files were updated when I removed `?view=office-js` from URLs throughout the doc set.)
